### ISSUE TITLE
prebuild files not fetched when using registry-proxy

### ISSUE THREAD

**Issue by mknj** [mapbox/node-pre-gyp#673]
When using a registry proxy the pre-build sqlite3 files are not fetched.
The problem is really annoying when installing sqlite3 in a ci via a docker image like node:16-alpine.

# steps to reproduce
## terminal 1 / start registry proxy
```
$ npm i verdaccio
$ ./node_modules/.bin/verdaccio
```
## terminal 2 / use proxy
```
$ npm i --registry=http://localhost:4873/ sqlite3
...
building from source
...
```

# expected result
the files were fetched from the cdn and no build happens

### workaround
```
mkdir hack;cd hack;npm -y init; npm i --registry=https://registry.npmjs.org/ sqlite3;mv node_modules ..;cd ..
npm i
```

**Comment by juanpicado** [comment]
It might be related with https://github.com/verdaccio/verdaccio/issues/3366

**Comment by mknj** [comment]
fixed in verdaccio@5.18.0
